standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -5919,7 +5919,7 @@ private void alter_partitions_with_environment_context(String catName, String db
 
     if (LOG.isInfoEnabled()) {
       for (Partition tmpPart : new_parts) {
-        LOG.info("New partition values:" + tmpPart.getValues());
+        LOG.info("New partition values: {}.{}.{}:{} ", catName, db_name, tbl_name, tmpPart.getValues());
```

Comment:
Can we put values like: catalog: {} database: {} table: {} partition: {} rather than using .